src/amo/components/SearchResult.js

```diff
@@ -28,11 +28,12 @@ export class SearchResultBase extends React.Component {
             <div className="SearchResult-rating">
               <Rating rating={addon.ratings.average} readOnly size="small" />
             </div>
-            <h3 className="SearchResult-author">{addon.authors[0].name}</h3>
-            <h3 className="SearchResult-users">{i18n.sprintf(
-              i18n.ngettext('%(total)s user', '%(total)s users', averageDailyUsers),
-              { total: i18n.formatNumber(averageDailyUsers) },
-            )}
+            <h3 className="SearchResult-author-users">
+              <span className="SearchResult-author">{addon.authors[0].name}</span>
```

Comment:
I tried this but its giving me a syntax error on-
`addonAuthor: addon.authors.length ? addon.authors[0].name : null,`
It says unexpected token ":"